inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -51,6 +51,9 @@ Parsed<T> parseDeviceNameIntoConfig(const std::string& deviceName, const std::ma
         deviceName_ = "MULTI";
         config_[InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES] = deviceName.substr(6);
     } else {
+        if (deviceName_ == "") {
+            deviceName_ = "AUTO";
```

Comment:
I think we also can add `deviceName = {}` in `ie_cor.hpp`

Reply:
yes, but maybe I am not suitable to change this API in this PR. :)